Makefile.d/test.mk

```diff
@@ -17,47 +17,47 @@
 .PHONY: test
 ## run tests for cmd, internal, pkg
 test:
-	go test -cover ./cmd/... ./internal/... ./pkg/...
+	go test -mod=readonly -cover ./cmd/... ./internal/... ./pkg/...
```

Comment:
should we add `-shuffle=on` option to go test?

Reply:
That is not the scope of this PR.
There are a lot of "go test" execution points in our project, it's a @vdaas/set task, and you should work on it.